tests/dummy/app/controllers/users.js

```diff
@@ -1,9 +1,10 @@
 import Controller from '@ember/controller';
 import { action } from '@ember/object';
+import { waitForPromise } from '@ember/test-waiters';
 
 export default class extends Controller {
   @action
   deleteUser(user) {
-    return user.destroyRecord();
+    waitForPromise(user.destroyRecord());
```

Comment:
ember-fetch provided a fetch that is registered with settled checks in tests. After dropping it, some tests started failing because they were acting on the records while they were still "inFlight"

So we need to use test-waiters for these, to ensure the test doesn't continue until the requests are settled